docs/home/benefits-of-pwa.md

```diff
@@ -0,0 +1,95 @@
+
+Re-uploading your app to native app stores is only necessary if you change your Web App Manifest or if you want to change the listing itself. New content can be delivered quickly and effectively without waiting on native app store processes.
+
+### Variety of Development Options
+
+The only requirement when developing a progressive web app is that you are using a technology that is capable of being deployed to the web. Because the web has been open and iterated on for so long, the number of development options is nearly endless. You can use thousands of frameworks on the front end to any language on the backend, and even use things like WebAssembly or WebGPU for high performance and critical code.
+
+PWABuilder advocates the use of Web Components to build out our resources, but any web framework you are comfortable with will work.
+
+If you want to get started with building a progressive web app with PWABuilder's recommended set of tooling, check out the documentation for the [PWA Starter.](/starter/quick-start/)
+
+## Users
+
+### Consistent Experience Across Platforms
+
+Applications developed for the web aren't as dependent on individual platforms requirements and specifications, and because of this, progressive web apps are great for creating a consistent experience across different platforms.
+
+A user who uses Windows for the desktop computer and Android for their mobile phone can install the exact same app to both of their devices, and have a familiar user experience even though the application is running on two entirely different platforms.
+
+### Accessible from Anywhere
+
+Another huge advantage is that even though progressive web apps can be installed to enable better functionality, they don't *need* to be. If a user doesn't have access to their personal device, they can access a favorite PWA of theirs from any device with browser capability.
+
+Additionally, PWAs also give the user freedom of choice - to be able to install an app or not, and still being able to make effective use of that app regardless of that choice.
+
+### Native Integration
+
+In the past, one of the main advantages native app development technology has had over web apps is the ability to integrate with the native platform easily and quickly with built-in APIs. However, with the advent of modern web technology, many integrations that were previously inaccessible from the web are now implemented in most modern Web Browsers (especially those built on Chromium, like Edge or Chrome itself).
+
+Native features that can be implemented from the web include, but are not limited to:
+
+- Push notifications
+- Background sync
+- Sharing files to and from your applications 
+- Custom file and URL handling
+- Integration with hardware devices, like touch screens, microphones, and other sensors
+
+You can learn about how to add native features to your PWA [here.](/home/native-features/)
```

Comment:
Hmm if this section is for users, learn how to integrate native features seems weird.

Reply:
Good point. Thanks.